webpack/entrypoints.js

```diff
@@ -81,6 +81,7 @@ const entrypoints = {
   ],
   "wiki.editor": [
     "sumo/js/wiki.editor.js",
+    "sumo/js/form-wizard.js",
```

Comment:
@escattone I added this since we need the element to be able to accurately preview the wizard when editing, but let me know if there's a different way we want to handle this.

Reply:
@hannajones Hm, I thought I had covered those additional places where the `switching-device.js` bundle should be loaded with the merge of https://github.com/mozilla/kitsune/pull/5462, so that you wouldn't need to do this, but I see that I missed the pages where the content can be previewed. Sorry about that.

I think we'd like to keep the JS for this only loaded for the pages that use your wizard (at least for now), so would you mind removing this and putting your `form-wizard.js` code within `device-migration-wizard.js`, and then I'll create a new PR that ensures that the `switching-device.js` bundle is loaded for all of the pages that allow the preview of content while editing? Does that make sense?

Reply:
That makes total sense, I reverted my change to `entrypoints.js`. Thanks for the clarification!

Reply:
@hannajones I created https://github.com/mozilla/kitsune/pull/5466 to cover the preview cases. Thanks again for catching those!